devel/libtool/Portfile

```diff
@@ -80,12 +80,9 @@ post-destroot {
 
     # libtool 2.4.6 contains com.dropbox.attributes xattrs,
     # let's avoid spreading them further (#54459)
-    # xattr on leopard or earlier does not support -r so do the recursion manually
-    if {${os.platform} eq "darwin" && ${os.major} >= 10} {
-        system "xattr -r -d com.dropbox.attributes ${destroot}"
-    } else {
-        system -W ${destroot} "find . -type f -print0 | xargs -0 xattr -d com.dropbox.attributes"
-    }
+    # xattr on Leopard or earlier does not support -r so do the recursion manually
+    # xattr on Monterey supports -r but errors on directories, so do fully manual
+    system -W ${destroot} "xattr -d com.dropbox.attributes \$(find . -type f -print0 | xargs -0 xattr | grep com.dropbox.attributes | awk '{ print \$1 }' | sed -e 's@:\$@@g')"
```

Comment:
Why did this have to become more complicated than the code from the previous else clause?

Reply:
I'm not in front of that build system at the moment, but memory is that not all files had that attribute & for those that didn't `xattr` errored on trying to remove the attribute. My version removes the attribute only from files where it is set. Worth checking to see if this & the above for directories with `xattr` is new expected behavior or a not (and hence a bug).